satpy/etc/readers/omps_edr.yaml

```diff
@@ -166,4 +170,74 @@ datasets:
     units: degrees
     standard_name: latitude
 
+  Longitude:
```

Comment:
The other datasets for longitude and latitude were named that way to avoid conflicts. Could we maybe name these in a similary way (maybe include `so2` and `pca`)?